airflow/plugins/operators/elavon_to_gcs_raw.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+import gcsfs
+import paramiko
+
+from airflow.models import BaseOperator
+
+CALITP__ELAVON_SFTP_PASSWORD = os.environ["CALITP__ELAVON_SFTP_PASSWORD"]
```

Comment:
can this be a secret rather than env var? to mirror GTFS and what I'm doing for Littlepay